wrangler-docs/functions/date-and-time-functions.md

```diff
@@ -0,0 +1,961 @@
+# Date and Time
+You can use the date and time functions to perform various operations on dates and times in the Wrangler.
+Functions that specify `dates`, `times`, or `timestamps` in the arguments use strings with specific formats:
+ - For a `date`, the format is `%yyyy-%mm-%dd` and is of java type `LocalDate`.
+ - For a `time`, the format is `%hh:%nn:%ss`. If extended to include microseconds, the format is `%hh:%nn:%ss.x` where
+ `x` gives the number of decimal places seconds is given to. `time` is of type `LocalTime`.
+ - For a `timestamp`, the format is the date format followed by the time format. `timestamp` is of type `LocalDateTime`.
+
+Functions that have the days of week in the argument take a string that specifies the day of the week.
+The day is specified as a three-letter abbreviation, or the full name. For example, the strings "mon" and "monday"
+are both valid.
+
+## CurrentDate
+Returns the `date` when the function is invoked.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+date (`date`)
+
+#### Example
+Use this function to add the current date to a column.
+```
+set-column date datetime:CurrentDate()
+```
+
+## CurrentTime
+Returns `time` when the function in invoked.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+time (`time`)
+
+#### Example
+Use this function to add the current time to a column.
+```
+set-column time datetime:CurrentTime()
+```
+
+## CurrentTimeMS
+Returns `time` in milliseconds when the function in invoked.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+time (`time`)
```

Comment:
these docs should all mention the type of the output and the type(s) of input. Should also mention what happens to null input.